app/containers/Home/Home.jsx

```diff
@@ -1,50 +1,143 @@
 
+import Button from '../../components/Button/Button'
+import TextInput from '../../components/Inputs/TextInput/TextInput'
+import SelectInput from '../../components/Inputs/SelectInput/SelectInput'
 import styles from './Home.scss'
+import neonLogo from '../../images/neon-logo-redesign.png'
+// Icons
+import Login from '../../images/icons/Login.svg'
+import Plus from '../../images/icons/Plus.svg'
+import Wallet from '../../images/icons/Wallet.svg'
 
-const Home = () => (
-  <div id='home'>
-    <Link to={ROUTES.LOGIN_LOCAL_STORAGE}>
-      <div className={classNames('linkBox', styles.linkBox)}>
-        Login using a saved wallet
-      </div>
-    </Link>
-    <Link to={ROUTES.LOGIN_NEP2}>
-      <div className={classNames('linkBox', styles.linkBox)}>
-        Login using an encrypted key
-      </div>
-    </Link>
-    <Link to={ROUTES.LOGIN_PRIVATE_KEY}>
-      <div className={classNames('linkBox', styles.linkBox)}>
-        Login using a private key
-      </div>
-    </Link>
-    <Link to={ROUTES.LOGIN_LEDGER_NANO_S}>
-      <div className={classNames('linkBox', styles.linkBox)}>
-        Login using a Ledger
-      </div>
-    </Link>
-    <Link to={ROUTES.CREATE_WALLET}>
-      <div className={classNames('linkBox', styles.linkBox, styles.linkBoxAlt)}>
-        Create a new wallet
-      </div>
-    </Link>
-    <Link to={ROUTES.ENCRYPT_KEY}>
-      <div className={classNames('linkBox', styles.linkBox, styles.linkBoxAlt)}>
-        Encrypt an existing key
-      </div>
-    </Link>
-    <Link to={ROUTES.SETTINGS}>
-      <div className={classNames('linkBox', styles.linkBox, styles.linkBoxAlt)}>
-        Manage Neon settings
+type State = {
+  option: string,
+  password: string
+}
+
+type Props = {
+  loginWithPrivateKey: Function
+}
+
+class Home extends Component<Props, State> {
+  state = {
+    option: 'Private Key',
+    password: ''
+  }
+
+  handleSelect = (option: string) => this.setState({ option })
+
+  handleSubmit = () => {
+    const { loginWithPrivateKey } = this.props
+    switch (this.state.option) {
+      case 'Private Key':
+        return loginWithPrivateKey(this.state.password)
+      default:
+        console.log('foo')
+    }
+  }
+
+  render = () => (
+    <div id="home" className={styles.home}>
+      <div className={styles.loginContainer}>
+        <img className={styles.logo} src={neonLogo} />
+        <div className={styles.loginText}>Login</div>
+
+        <div className={styles.inputContainer}>
+          <SelectInput
+            className={styles.input}
+            onChange={value => this.handleSelect(value)}
+            value={this.state.option}
+            items={['Private Key']}
+            getItemValue={() => this.state.option}
+          />
+
+          <TextInput
+            className={styles.input}
+            onChange={(e: Object) =>
+              this.setState({ password: e.target.value })
+            }
+            placeholder="Password"
+            type="password"
+          />
```

Comment:
I would recommend moving the contents of the input container into its own component.  That way, based upon the value of `this.state.option`, you can just render a single component.  You could expose an `onLogin` prop for when the user submits there password (for passphrase login) or any other data (for other login methods).